python/nano/tutorial/notebook/inference/pytorch/pytorch_save_and_load_jit.ipynb

```diff
@@ -8,20 +8,25 @@
       ]
     },
     {
+      "attachments": {},
       "cell_type": "markdown",
       "metadata": {},
       "source": [
         "# Save and Load Optimized JIT Model\n",
         "\n",
         "This example illustrates how to save and load a model accelerated by JIT.\n",
-        "In this example, we use a ResNet18 model pretrained. Then, by calling `InferenceOptimizer.trace(model, accelerator=\"jit\"...)`, we can obtain a model accelarated by JIT method. By calling `InferenceOptimizer.save(model_name, path)` , we could save the model to a folder. By calling `InferenceOptimizer.load(path)`, we could load the model from a folder."
+        "\n",
+        "In this example, we use a pretrained ResNet18 model. Then, by calling `InferenceOptimizer.trace(model, accelerator=\"jit\"...)`, we can obtain a model accelarated by JIT method. By calling `InferenceOptimizer.save(model_name, path)` , we could save the model to a folder. By calling `InferenceOptimizer.load(path)`, we could load the model from a folder."
```

Comment:
well, some optimizations must be loaded with `InferenceOptimizer.load(path, original_model)`

Reply:
Have updated to show that IPEX needs original model here.